tests/integration/netapi/rest_tornado/test_app.py

```diff
@@ -203,8 +203,8 @@ def test_multi_local_async_post(self):
         self.assertEqual(len(ret), 2)
         self.assertIn('jid', ret[0])
         self.assertIn('jid', ret[1])
-        self.assertEqual(ret[0]['minions'], sorted(['minion', 'sub_minion', 'localhost']))
-        self.assertEqual(ret[1]['minions'], sorted(['minion', 'sub_minion', 'localhost']))
```

Comment:
Why are these getting removed?

This was added for testing the `enable_ssh_minions` feature added in 2018.3.

It does need the ssh daemon started in runtests.py with `python -m tests.runtests --ssh`, but those minions should still be returning from the tornado tests.

Reply:
OK, this is what I needed to know, thanks.